docs/wasmer-js/node-modules/examples/hello-world/hello-world.md

```diff
@@ -6,113 +6,166 @@ sidebar_label: Hello World
+> ### Mac users
+> Before the installation of Parcel will work on a Mac, you must first install the [Xcode Command Line Tools](https://developer.apple.com/download/more/?=for%20Xcode)
 
-`npm install --save-dev parcel-bundler parcel-plugin-static-files-copy`
+## Run Example from Git Clone
 
-Let's add some files to ensure parcel is working! First, let's add the `index.html` shown one their homepage:
+The simplest way to run this exercise is to clone the entire [`docs.wasmer.io`](https://github.com/wasmerio/docs.wasmer.io) repo:
 
-```html
-<html>
-<body>
-  <script src="./index.js"></script>
-</body>
-</html>
-```
+1. Change into some development directory
 
-Then, let's add our `index.js` script tag:
+    ```bash
+    $ cd <some_development_directory>
+    ```
 
-``console.log('I am working')``
+1. Clone this entire repo
 
-Let's serve our project so far using the parcel CLI:
+    ```bash
+    $ git clone https://github.com/wasmerio/docs.wasmer.io.git
+    ```
 
-`parcel index.html`
+1. Change into the `hello-world` example directory
 
-If we open the URL in the browser like parcel suggests, `http://localhost:1234`, we should get a blank page. But if we open the JavaScript console, we should see "I am working" logged, meaning everything is working!
+    ```bash
+    $ cd docs.wasmer.io/docs/wasmer-js/node-modules/examples/hello-world
+    ```
 
-Now that we got our Javascript project set up, let's go ahead and add the following Wasmer-JS packages: 
+1. Install the required `npm` dependencies
 
-- `@wasmer/wasi` - which will act as a polyfill for wasi bindings that the browser does not support.
-- `@wasmer/wasm` - which provides a sandboxed filesystem that our wasi module will interact with
+    ```bash
+    $ npm install
+    ```
 
-To install these packages as dependencies to the project, run the following:
+1. Start `parcel` 
 
-`npm install --save @wasmer/wasi @wasmer/wasmfs`
+   ```bash
+   $ parcel index.html
+   ```
 
-Now that these packages are installed, let's change our index.js to the following:
+1. Point your browser to [`http://localhost:1234`](http://localhost:1234) and you should see `Standard Output: Hello World!` appear both on the browser screen and in the JavaScript console
 
-```javascript
-// Imports
-import { WASI } from '@wasmer/wasi';
-import { WasmFs } from '@wasmer/wasmfs';
 
-// The file path to the wasi module we want to run
-const wasmFilePath = './as-echo.wasm';
+## JavaScript Coding
+
+Seeing as this is demo code, it uses meaningful variable names and contains lots of explanatory comments (features that are often sadly missing from production code).  Please take some time to read and understand these comments as they explain how the functionality has been constructed.
+    
+Also, make a note of the comment explaining `@wasmer/wasm-transformer`; we will cover this very important detail in a later example.
+
+
+```JavaScript
+// *****************************************************************************
```

Comment:
Well, to avoid going round in circles on this subject, when creating code specifically for the purpose of education, I've found that it helps students if there is some sort of visual separator between the different blocks of code.

How visually intrusive these separators are can be controlled by the choice of character:

* A line of asterisks stands out the strongest
    ```// *******************************```

* A line of equals signs is slightly less intrusive
    ```// ===============================```

* A line of minus signs is less intrusive
    ```// -------------------------------```

* A line of alternating minus signs and spaces is the least intrusive
    ```// - - - - - - - - - - - - - - - -```

If it doesn't bug you too much, I'd like to leave these separators in, but downgrade them to the least visually intrusive...  🙏🏻

Reply:
@ChrisWhealy Ah, well if you've found them to be helpful in education in the past, we can totally keep them 😄